plugin/storage/cassandra/schema.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Not quite sure if this is the most appropriate for this script. It has to be either on the same directory as the Dockerfile or in a sub directory, but not on a parent directory.